### DISCUSSION TITLE
I need help calibrating my current sensor!!

### DISCUSSION THREAD

**Discussion by doodi0507** [iNavFlight/inav#6673] [Q&A]
have a the Zeus f760 stack (fc - f722) on a 7 inch frame with br2297s 1600kv motor with 6inch props. The issue is that my battery meter on my osd says it’s too low when I take off and my quad falls to the ground. I tried calibrating the battery but maybe I did it incorrectly. GPS and everything else works fine I’m just having a issue with keeping it in the air. I have flight footage but it’s mostly at night because I only did it for data! I’m new to the hobby but not new to soldering and a little coding. With the props off the battery ran for about 8 or 9 minutes. I also built the battery from li-ion cells and they read 24v when fully charged (6s) 1800mah!  is it that I don’t have enough capacity. I could use a sponser or something that I can ask these types of questions when I get stuck. If your interested let me know. I’m not ashamed to admit I need help sometimes!! Flight footage on my profile in the repository the wasp issues tab!! Please help.

**Comment by breadoven** [comment]
How much current is it drawing ? Li-ions can't handle high currents, the voltage sags badly even at currents well below the stated max for the cells.

**Reply by doodi0507** [reply]
When I was testing it I fried a capacitor in my esc. I have a new one on the way and a lipo so I can test it again. I'll give an update when I get it. Thank you for the response!!

**Comment by DzikuVx** [comment]
If it `The issue is that my battery meter on my osd says it’s too low when I take off and my quad falls to the ground.` then it's not calibration but the battery is damaged/broken/old/whatever. INAV can not make quad fell from the sky only because current sensor says battery is depleted.
battery really is depleted!
